Vinyl/Response.swift

```diff
@@ -60,9 +60,9 @@ func ==(lhs: Response, rhs: Response) -> Bool {
 extension Response: Hashable {
     
     var hashValue: Int {        
-        let body = self.body == nil ? "\(self.body)" : ""
-        let error = self.error == nil ? "\(self.error)" : ""
+        let body = self.body == nil ? "" : "\(self.body!)"
```

Comment:
It's force unwrapping in a context where `self.body` is guaranteed to be not nil. What alternative would you suggest? Incidentally, the previous code was broken as the logic was the opposite of what was intended.